asset_dashboard/models.py

```diff
@@ -700,39 +700,36 @@ class Meta(GISModel.Meta):
     remarks = models.TextField()
     photos = models.TextField()
 
-
 class ParkingLots(GISModel):
-    """Parking lot polygons, for all public and non-public lots."""
-
     class Meta(GISModel.Meta):
-        db_table = '"quercus"."parking_lots"'
-
-    id = models.AutoField(primary_key=True, db_column='parking_lots_id')
+        db_table = '"acer"."parking_lots_union_mv"'
```

Comment:
query this materialized view in the `acer` table instead of the `quercus` table.